core/jvm/src/test/scala-2.12/zio/StacktracesSpec.scala

```diff
@@ -345,7 +345,7 @@ class StacktracesSpec(implicit ee: org.specs2.concurrent.ExecutionEnv)
       (cause.traces must have size 1) and
         (cause.traces.head.executionTrace must mentionMethod("traceThis")) and
         (cause.traces.head.executionTrace must not have mentionedMethod("tracingRegions")) and
-        (cause.traces.head.stackTrace must beEmpty)
+        (cause.traces.head.stackTrace must have size 1)
```

Comment:
Is this ok (it's hard for me to tell why it changed)?

Reply:
Since there's a new `fold` after release it will end up in the stack trace when release fails, that's correct and desirable. (In `release(a, e).fold(x)` the `x` is waiting on stack for the value of release)

Reply:
actually, no, the error happens in use, not in release, it's a bug in how `.traced` is implemented on top bracket. It worked before because `release(a, e) *> IO.done(e)` was in FlatMap fast path

Reply:
Addressed in https://github.com/zio/zio/pull/1047